x-pack/plugins/lists/common/schemas/request/update_list_schema.mock.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { DESCRIPTION, LIST_ID, META, NAME, _VERSION } from '../../constants.mock';
+
+import { UpdateListSchema } from './update_list_schema';
+
+export const getUpdateListSchemaMock = (): UpdateListSchema => ({
+  _version: _VERSION,
+  description: DESCRIPTION,
+  id: LIST_ID,
+  meta: META,
+  name: NAME,
+});
+
+/**
+ * Useful for end to end tests and other mechanisms which want to fill in the values
```

Comment:
This comment might be useful in the mocks added in `create_list_schema.mock.ts` too?

Reply:
Yeah, let me change and add it there.